<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerPageClass</class>
 <widget class="QMainWindow" name="ManagerPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagerPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/GymPadelSystem/Resources/Gym.png&quot;);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="LWidget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>731</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						background: none;
						background-color: rgba(0,0,0,0.7);
						border-radius: 25px;
					</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>320</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							font-size: 30px;
							color: rgba(135, 206, 235,0.8)
						</string>
     </property>
     <property name="text">
      <string>Manager Dashboard</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>520</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							background-color:rgba(0,0,0,0.7) ;
							color:rgba(135, 206, 235,0.8);
						</string>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>First Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Middle Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visits</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>400</y>
       <width>151</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							font-size: 20px;
							color: rgba(135, 206, 235,0.8)
						</string>
     </property>
     <property name="text">
      <string>Total Revenue </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="TRtextBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color:rgba(0,0,0,0.7) ;
color: rgba(135, 206, 235,0.8)
</string>
     </property>
     <property name="text">
      <string>4400</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						QPushButton
						{
						background: none;
						background-color: rgba(255, 51, 51,0.9);
						color: white;
						border-radius: 25px;
						}
						QPushButton:hover
						{
						border: 2px solid rgba(255, 51, 51,0.9);
						background-color: rgba(250, 250, 250,0.8);
						color: rgb(255, 51, 51);
						}
					</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
